cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -54,6 +76,27 @@ checkoutConfiguration.paymentMethodsConfiguration = {
             firstName: document.getElementById("shippingFirstNamedefault").value,
             lastName: document.getElementById("shippingLastNamedefault").value
         }
+    },
+    paypal: {
+        intent: "capture",
+        onSubmit: (state, component) => {
+            assignPaymentMethodValue();
+            document.querySelector("#adyenStateData").value = JSON.stringify(componentArr[selectedMethod].stateData);
+            makePaypalPayment(state.data, component);
```

Comment:
can we make this more genereric as we have multiple payment methods where we first need to payments call and then the follow up call think about QR codes rendering. Would be great to provide generic solution for them all.